docs/crio.conf.5.md

```diff
@@ -25,143 +23,188 @@ No bare options are used. The format of TOML can be simplified to:
     option = value
 
 ## CRIO TABLE
+CRI-O reads its storage defaults from the containers-storage.conf(5) file located at /etc/containers/storage.conf. Modify this storage configuration if you want to change the system's defaults. If you want to modify just CRI-O, you can change the storage configuration options here.
 
-The `crio` table supports the following options:
+**root**="/var/lib/containers/storage"
+  Path to the "root directory". CRI-O stores all of its data, including containers images, in this directory.
 
+**runroot**="/var/run/containers/storage"
+  Path to the "run directory". CRI-O stores all of its state in this directory.
 
-**root**=""
-  CRIO root dir (default: "/var/lib/containers/storage")
+**storage_driver**="overlay"
+  Storage driver used to manage the storage of images and containers. Please refer to containers-storage.conf(5) to see all available storage drivers.
 
-**runroot**=""
-  CRIO state dir (default: "/var/run/containers/storage")
+**storage_option**=[]
+  List to pass options to the storage driver. Please refer to containers-storage.conf(5) to see all available storage options.
```

Comment:
I'm not seeing "all available" options/drivers in my man page for containers-storage.conf.  Is it out of date and/or will be updated?

Reply:
We should update it.

Reply:
I opened https://github.com/containers/storage/issues/209 for that and optimistically made the that statement already here :-D